go/cmd/betabot/main.go

```diff
@@ -58,63 +60,136 @@ func betabot() error {
 	}
 
 	// event loop
+	var wg sync.WaitGroup
+	wg.Add(1)
```

Comment:
To avoid confusion we usualy try to put the `wg.Add()` right before firing the goroutine (so it's easy to connect the Add and the goroutine) but that just styling, this work just fine.

EDIT: And here that needed, lets assume `messengerClient.EventStream` return an error you are never gonna do a `Done` (because the return stop the current scope never reaching the `go func(){defer wg.Done() // ...`)